docs/www/nodule-grid.js

```diff
@@ -377,6 +377,368 @@
+                        "Finding": "Pulmonary nodule",
+                        "Attenuation Characteristic": "Solid",
+                        "Radiographic Lesion Margin": "Lesion with circumscribed margin",
+                        "Finding site": "Upper lobe of right lung",
+                        "Lung-RADS assessment": "Lung-rads 2",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "6.1",
+                        "Maximum 3D diameter": "6.2",
+                        "Maximum perpendicular 2D diameter": "6.1",
+                        "Mean 2D diameter": "6.1",
+                        "Volume": "42.1"
+                    },
+                    {
+                        "Tracking Identifier": "R8",
+                        "Tracking Unique Identifier": "1.3.12.2.1107.5.8.21.192168059.30000021092410281563200001347",
+                        "Finding": "Pulmonary nodule",
+                        "Attenuation Characteristic": "Unknown",
+                        "Radiographic Lesion Margin": "Lesion with circumscribed margin",
+                        "Finding site": "Lower lobe of right lung",
+                        "Lung-RADS assessment": "Lung-rads 0",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.5",
+                        "Maximum 3D diameter": "6.2",
+                        "Maximum perpendicular 2D diameter": "4.2",
+                        "Mean 2D diameter": "4.9",
+                        "Volume": "34.5"
+                    },
+                    {
+                        "Tracking Identifier": "R9",
+                        "Tracking Unique Identifier": "1.3.12.2.1107.5.8.21.192168059.30000021092410281563200001347",
+                        "Finding": "Pulmonary nodule",
+                        "Attenuation Characteristic": "Solid",
+                        "Radiographic Lesion Margin": "Lesion with circumscribed margin",
+                        "Finding site": "Upper lobe of right lung",
+                        "Lung-RADS assessment": "Lung-rads 2",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.4",
+                        "Maximum 3D diameter": "5.4",
+                        "Maximum perpendicular 2D diameter": "4.0",
+                        "Mean 2D diameter": "4.7",
+                        "Volume": "21.4"
+                    }
+                ];
+
+                //VAPALS.displayNotification("Updating Data");
+                if (!confirm("Do you really want to do the data update?")) {
+                    return;
+                }
+
+                // if number of nodules shown is less than the number of entries in theData array then increase the number of nodules to they match
+                while (settings.getNoduleCount() < theData.length) {
```

Comment:
Instead of this looping, just set noduleCount = data.length, then call `_displayNodules(noduleCount)`.